src/versions/2.0/json.ts

```diff
@@ -96,6 +132,17 @@ const STANDARD_CHARGE_DEFINITIONS = {
       },
     },
     required: ["payer_name", "plan_name", "methodology"],
+
+    if: {
+      properties: {
+        methodology: {
+          const: "other",
+        },
+      },
+    },
```

Comment:
It would be useful to include `required: [ "methodology" ]` as part of this subschema. That way, if `methodology` is not in the document, the subschema will fail, and the `then` subschema won't be applied. Adding in this requirement gets rid of the two extra errors that appear in the test for `sample-errors.json`.

Reply:
Thanks! Updated.